assets/js/components/ChecksCatalog/ChecksCatalogNew.jsx

```diff
@@ -1,66 +1,81 @@
 /* eslint-disable react/no-array-index-key */
-import React, { useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 
 import { groupBy } from '@lib/lists';
 
 import { getCatalog } from '@state/selectors/catalog';
 import { updateCatalog } from '@state/actions/catalog';
+import { providersList } from '../ProviderLabel/ProviderLabel';
 import CatalogContainer from './CatalogContainer';
 import CheckItem from './CheckItem';
+import ProviderSelection from './ProviderSelection';
 
+const providers = ['default'].concat(providersList.slice());
```

Comment:
I think the labels approach is wrong, as you would be sending the labels in the query, and we might not recognize them. We would need to send the keys:
We don't want to send `provider=AWS` we want to send `provider=aws`